scripts/deploy-helm.sh

```diff
@@ -51,41 +51,6 @@ check_dynatrace_log_ingest_url() {
   fi
 }
 
-check_activegate_state() {
```

Comment:
We are already checking connection (even sending a test log line through it) to provided ingestLogsUrl